### DISCUSSION TITLE
Airpackage format used by Distriqt ANEs

### DISCUSSION THREAD

**Discussion by Michoko92** [airsdk/Adobe-Runtime-Support#3346] [Q&A]
Hi,

I wanted to update some ANEs from Distriqt, but on their Github repo, all releases are provided with ".airpackage" filename extension. I'm still using FlashDevelop, which works perfectly fine for me. Until now, I used to add ANE files directly to my project's library, but airpackages are not recognized by FD.

I found out that I could open an airpackage with 7zip, then extract the ANE files from it. But do you know what's the purpose of the airpackage format, and can it be used directly by the AIR SDK? 

Thank you!

**Comment by ghost** [comment]
Adding and updating ane's manually is a pain. I realize it after I start to use apm. Give it a try.

**Reply by marchbold** [reply]
Glad you are finding it useful! :)

**Comment by marchbold** [comment]
@Michoko92 All our extensions are still available as ane files in the repositories at the same location as they have always been? You definitely don't need to unzip the airpackage files. 

The `airpackage` files are published as assets in the repository for use with `apm`, the [AIR package manager](https://github.com/airsdk/apm). You don't **have** to use `apm` but we highly recommend it. 

It simplifies installation to a command like:

```
apm install com.distriqt.Notifications
``` 

And updating the manifest / info additions:

```
apm generate app-descriptor
```

You can check the install guide [here](https://airsdk.dev/docs/basics/install-apm)

**Reply by Michoko92** [reply]
Thank you Michael. I've been a long-time user of your ANEs, but always handled them manually, so I had missed the apm installation procedure. Keep up the great work!

**Reply by marchbold** [reply]
If you are looking to use apm I wrote an introduction here

https://github.com/airsdk/apm/discussions/31

But do let me know if you can't find the ane's in the repository
